pkg/operator/operator.go

```diff
@@ -49,6 +50,10 @@ var (
 	log = logf.Logger.WithName("init")
 )
 
+const (
+	IgnoreAutoCreateDefaultIngressControllerEnvVar = "IGNORE_AUTO_CREATE_DEFAULT_INGRESS_CONTROLLER"
```

Comment:
*nit* this might just be me, but the verb IGNORE feels a bit awkward here. Would using DISABLE make more sense?

Also, the word "AUTO" implies to me that, we can still somehow manually tell the ingress-operator to manage the default ingress controller, but I don't think that's true. I think you are disabling the ingress-operator from managing the default ingress controller completely right?

Would something like `DISABLE_DEFAULT_INGRESS_CONTROLLER=true` be more simple and explicit?

Reply:
yep, makes sense